iceoryx_hoofs/source/posix_wrapper/unix_domain_socket.cpp

```diff
@@ -523,7 +522,7 @@ IpcChannelError UnixDomainSocket::convertErrnoToIpcChannelError(const int32_t er
 
 bool UnixDomainSocket::isNameValid(const UdsName_t& name) noexcept
 {
-    return !(name.empty() || name.size() < SHORTEST_VALID_NAME || name.size() > LONGEST_VALID_NAME);
```

Comment:
Why is this removed?

Reply:
There is only the requirement that you provide some kind of filename, therefore the name is not allowed to be empty. The `UdsName_t` is an alias for `cxx::string<LONGEST_VALID_NAME>` therefore the greater than check is done via the type.

A socket file name with one letter is allowed. Therefore we can reduce the check to `name.empty()`.

Reply:
So `"/"` is a valid name ;)

Reply:
Of course ... ;) I think in Windows it actually is maybe?!.
But I can extend the test that only the following characters are allowed `[a-zA-Z0-9.,-]` but it wasn't there in the first place...

Reply:
You touched it ;)

That check is not sufficient since you can have `/tmp/mysocket`. The previous check was also not sufficient since this should also not be allowed `/////`. Maybe we just need to check if the last character isn't a path separator?

Reply:
... oh, and we need tests for this

Reply:
It seems this will be fixed with the next PR

Reply:
Is already fixed with tests n stuff in the following PR: https://github.com/eclipse-iceoryx/iceoryx/pull/820